source/clear-linux/tutorials/telemetry-backend/Telemetry-backend.rst

```diff
@@ -0,0 +1,495 @@
+visualize the telemetry data. The :guilabel:`telemetryui` app also provides a
+REST API to perform queries on the data.
+
+The applications run within a web stack, using the :guilabel:`nginx` web
+server, the :guilabel:`uWSGI` application server, and
+:guilabel:`PostgreSQL` as the underlying database server. For a detailed
+description, visit the `Clear Linux telemetry backend server overview`_.
+
+This tutorial will walk you through creating a telemetrics backend server on
+your local |CL| machine using the :command:`deploy.sh` bash shell script that
+is maintained in a GitHub repository at
+https://github.com/clearlinux/telemetrics-backend. Once the backend server is
+up and running we show you how to redirect telemetry records from the system
+you set up to this new server location.
+
+.. note::
+   The telemetrics functionality adheres to `Intel’s privacy policies`_
+   regarding the collection and use of Personally Identifiable Information
+   (PII) and is Open Source. Specifically, no intentionally identifiable
+   information about the user or system owner is collected.
+
+Prerequisites
+=============
+
+For this tutorial, start with a clean installation of |CL| on a new system
+using the :ref:`bare-metal-install` getting started guide:
+
+#. Choose to install |CL|
+#. Join the :guilabel:`Stability Enhancement Program` to install and
+   enable the telemetrics components
+#. Select the manual installation method with the following settings:
+
+   * Set the hostname to :guilabel:`clr-telem-server`,
+   * Create an administrative user named :guilabel:`clear` and add this user
+     to sudoers,
+   * Add all additional software bundles
+
+Optionally, if you already have a |CL| system set up that you want to install
+the telemetry backend server components to, or to a virtual |CL| machine, you
+need to have a user account that has sudo privileges and the following
+software bundles already installed:
+
+* dev-utils - has the :command:`git` command
+* telemetrics - has the required telemetry applications, see the 
+  :ref:`telemetry-enable` guide for installing and enabling telemetrics.
+
+To determine the software bundles already installed, enter the
+:command:`sudo swupd bundle-list` command to list the current bundles
+installed and then run :command:`sudo swupd bundle-add dev-utils telemetrics`
+if either one is not installed.
+
+Download the clearlinux/telemetrics-backend git repository
+==========================================================
+
+With all prerequisite software bundles installed and logged in with your
+administrative user, from your $HOME directory, run :command:`git` to clone
+the :guilabel:`telemetrics-backend` repository into the
+:file:`$HOME/telemetrics-backend` directory:
+
+.. code-block:: console
+
+   git clone https://github.com/clearlinux/telemetrics-backend
+
+.. note::
+   You may need to set up the :guilabel:`https_proxy` environment variable
```

Comment:
Please use the envvar role for environment variables. http://www.sphinx-doc.org/en/stable/markup/inline.html?highlight=envvar

Reply:
done